tests/integration_tests/cloudbreak-workload/recordings/test_cloudbreak_integration_tests.test_b_workload_nsg_creation.yaml

```diff
@@ -75,7 +75,7 @@ interactions:
           policyclient/2.1.0 Azure-SDK-For-Python]
       accept-language: [en-US]
     method: PUT
-    uri: https://management.azure.com/subscriptions/00000000-0000-0000-0000-000000000000/providers/Microsoft.Authorization/policyDefinitions/permittedRegions?api-version=2018-03-01
+    uri: https://management.azure.com/subscriptions/00000000-0000-0000-0000-000000000000/providers/Microsoft.Authorization/policyDefinitions/permittedRegions?api-version=2018-05-01
```

Comment:
Can we have a global variable for api-version? For future changes, we can change the api version in one place.

Reply:
Good call, let's see if we face another instance like this, what we might need instead, is to configure Azure Python SDK to use the latest api-version if possible.